README.md

```diff
@@ -1,2 +1,76 @@
-# X-NUCLEO-IKS02A1-Audio
-X-NUCLEO-IKS02A1 audio library to support IMP34DT05 MEMS digital omnidirectional microphone 
+# X-NUCLEO-IKS02A1 Audio
+Arduino library to support the [IMP34DT05](https://www.st.com/en/mems-and-sensors/imp34dt05.html) digital microphone.
+
+## Setting SAI & I2S
+Up to [STM32 core](https://github.com/stm32duino/Arduino_Core_STM32) version 2.0.0 it is necessary to apply the following steps:
+
+Check where the Arduino `preferences.txt` file are in `Arduino IDE -> Preferences`.
+
+	For Example: `C:\Users\Duino\AppData\Local\Arduino15` (Windows)
+
+Then go to `..\packages\STM32\hardware\stm32\<release version>\cores\arduino\stm32\stm32yyxx_hal_conf.h` and open the file.
+
+In the file add these lines of code:
+
+```C
+	#if !defined(HAL_I2S_MODULE_DISABLED)
+  		#define HAL_I2S_MODULE_ENABLED
+	#else
+	  #undef HAL_I2S_MODULE_ENABLED
+	#endif
+
+	#if !defined(HAL_SAI_MODULE_DISABLED)
+	  #define HAL_SAI_MODULE_ENABLED
+	#else
+	  #undef HAL_SAI_MODULE_ENABLED
+	#endif
+```
+
+## API
+This library acquires PDM, converts PDM to PCM and stores the data on SD Card as WAVE file.
```

Comment:
Hi @sarachiodo ,
actually the SD card and wave stuff have been moved on the FP_Examples, so we can say only:

This library acquires PDM and converts PDM to PCM